test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
```

Comment:
remove this import statement since it is not used, it makes the linter fail

Reply:
Done